libs/documentation/src/lib/pages/layout/layout.component.ts

```diff
@@ -0,0 +1,238 @@
+import { Component, AfterViewInit, ViewChild } from '@angular/core';
+import { FormlyFieldConfig } from '@ngx-formly/core';
+import { BehaviorSubject } from 'rxjs';
+import { FormGroup } from '@angular/forms';
+import { CdkAccordionItem } from '@angular/cdk/accordion';
+import { DataService } from './data.service';
+import { SideNavigationModel } from '@gsa-sam/components';
+import { navigationConfig } from './navigate.config';
+import {
+  SearchListConfiguration,
+  SearchParameters
+} from '@sam-design-system/layouts';
+
+@Component({
+  templateUrl: './layout.component.html'
+})
+export class ResultsLayoutComponent implements AfterViewInit {
+  public navigationModel: SideNavigationModel = navigationConfig;
+  @ViewChild('resultList') resultList;
+  form = new FormGroup({});
+  filterModel = {};
+  filtersExpanded: boolean = true;
+  searchParameters: SearchParameters; // = new SearchParameters();
+  public filterChange$ = new BehaviorSubject<object>(null);
+  allDomainsListConfig: SearchListConfiguration = {
+    defaultSortValue: 'relevanceDescending',
+    pageSize: 25,
+    sortList: [
+      { text: 'Relevance', value: 'relevanceDescending' },
+      { text: 'Entity Name', value: 'legalBusinessName' },
+      { text: 'Status', value: 'registrationStatus' }
+    ]
+  };
+
+  public fields: FormlyFieldConfig[] = [

```

Comment:
Can we move to the structure that Christy uses, she has a service (and a base service sometimes) that handles the layout config, filters, model, etc.

Reply:
Updated to one service to make it simple